iceoryx2-pal/posix/build.rs

```diff
@@ -20,13 +20,9 @@ fn main() {
     #[cfg(any(target_os = "linux", target_os = "freebsd"))]
     println!("cargo:rustc-link-lib=pthread");
 
-    if std::env::var("DOCS_RS").is_ok() {
-        println!("cargo:rerun-if-changed=src/c/posix_docs_rs.h");
```

Comment:
This was obsolete code which is overlooked. The docs rs issue is addressed with the macro `IOX2_DOCS_RS_SUPPORT` which is defined below.